front/app/utils/projectUtils.ts

```diff
@@ -23,3 +25,12 @@ export function isFieldEnabled(
 
   return true;
 }
+
+export function isPhaseActive(phase: IPhaseData): boolean {
+  return (
```

Comment:
This looks cleaner that using pastPresentOrFuture directly, so I've added it as a new util.